### PR TITLE
Refactor the project.properties list

### LINKED ISSUE #164: Copy  new projects to project.properties list from contribution repo
Related to https://github.com/checkstyle/contribution/pull/895#issuecomment-2305913445

### PR DESCRIPTION
**Pull request by relentless-pursuit**
Resolves #164

### CODE REVIEW

**Comment by romani** [comment]
Please split into two commits, one in main change, second is results of generation.

In main change there should be change only to add two more lines, nothing else.

**Comment by relentless-pursuit** [comment]
> Please split into two commits, one in main change, second is results of generation.
> 
> In main change there should be change only to add two more lines, nothing else.

done.

**Comment by relentless-pursuit** [comment]
> Please split into two commits, one in main change, second is results of generation.
> 
> In main change there should be change only to add two more lines, nothing else.

Maybe, i copy pasted the entire proj.prop from that contribution. So, i had to just add the new projects?

File: extractor/src/main/resources/list-of-projects.properties [hunk 1]
```diff
@@ -3,37 +3,45 @@
 # Please note that bash comments works in this file
 
 # Few projects that delivers set of unusual Java constructions that shall be correctly handled by AST visitor
-# 'InputAllEscapedUnicodeCharacters' must be skipped because it is too big and slows down JXR
-checkstyle|git|https://github.com/checkstyle/checkstyle.git|master|**/.ci-temp/**/*,**/resources-noncompilable/**/asttreestringprinter/**/*,**/resources-noncompilable/**/filefilters/**/*,**/resources-noncompilable/**/main/**/*,**/resources-noncompilable/**/suppressionsstringprinter/**/*,**/resources-noncompilable/**/gui/**/*,**/resources-noncompilable/**/javadocpropertiesgenerator/**/*,src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/javaparser/InputJavaParser.java,**/InputAllEscapedUnicodeCharacters.java,**/resources-noncompilable/**/javaparser/InputJavaParser.java,**/resources-noncompilable/**/checks/imports/unusedimports/InputUnusedImportsSingleWordPackage.java,**/resources-noncompilable/**/grammar/java19/*,**/resources-noncompilable/**/treewalker/**/*
+checkstyle|git|https://github.com/checkstyle/checkstyle.git|master|
```
Comment (romani) [inline extractor/src/main/resources/list-of-projects.properties:6]: revert

File: extractor/src/main/resources/list-of-projects.properties [hunk 2]
```diff
@@ -3,37 +3,45 @@
...
 guava|git|https://github.com/google/guava|v28.2||
 
 spotbugs|git|https://github.com/spotbugs/spotbugs|3.1.2||
-pmd|git|https://github.com/pmd/pmd|pmd_releases/6.21.0|**/pmd/pmd-java/src/test/**/*,**/pmd/cpd/files/*
-spoon|git|https://github.com/INRIA/spoon.git|spoon-core-10.1.0|**/src/test/resources/**/*
-lombok-ast|git|https://github.com/rzwitserloot/lombok.ast|v0.2|**/lombok-ast/test/**/*
+pmd|git|https://github.com/pmd/pmd|pmd_releases/6.21.0|
+spoon|git|https://github.com/INRIA/spoon.git|spoon-core-8.0.0|
+lombok-ast|git|https://github.com/rzwitserloot/lombok.ast|v0.2|
```
Comment (romani) [inline extractor/src/main/resources/list-of-projects.properties:15]: revert

File: extractor/src/main/resources/list-of-projects.properties [hunk 3]
```diff
@@ -3,37 +3,45 @@
...
 
 spring-framework|git|https://github.com/spring-projects/spring-framework|v4.1.6.RELEASE||
-hibernate-orm|git|https://github.com/hibernate/hibernate-orm|4.2.19.Final|**/hibernate-orm/documentation/**/*
+hibernate-orm|git|https://github.com/hibernate/hibernate-orm|4.2.19.Final|
```
Comment (romani) [inline extractor/src/main/resources/list-of-projects.properties:18]: revert

File: extractor/src/main/resources/list-of-projects.properties [hunk 4]
```diff
@@ -3,37 +3,45 @@
...
 elasticsearch|git|https://github.com/elastic/elasticsearch|v1.5.2||
 java-design-patterns|git|https://github.com/iluwatar/java-design-patterns|dd855a376bc025aa61f6816584f79eb9854fe5d7||
 MaterialDesignLibrary|git|https://github.com/navasmdc/MaterialDesignLibrary|1.3||
 Hbase|git|https://github.com/apache/hbase|1.1.0.1||
 Orekit|git|https://github.com/CS-SI/Orekit|8.0.1||
 
 # Those projects are quite old and have lot of legacy code
-apache-ant|git|https://github.com/apache/ant|ANT_194|**/apache-ant/src/tests/**/*,**/apache-ant/src/etc/testcases/
+apache-ant|git|https://github.com/apache/ant|ANT_194|
```
Comment (romani) [inline extractor/src/main/resources/list-of-projects.properties:26]: revert

File: extractor/src/main/resources/list-of-projects.properties [hunk 5]
```diff
@@ -3,37 +3,45 @@
...
 apache-jsecurity|git|https://github.com/apache/jsecurity|c2ac5b90a467aedb04b52ae50a99e83207d847b3||
 android-launcher|git|https://github.com/android/platform_packages_apps_launcher|android-2.1_r2.1p2||
-apache-struts|git|https://github.com/apache/struts.git|master|**/apache-struts/**/resources/**/*
+apache-struts|git|https://github.com/apache/struts.git|master|
```
Comment (romani) [inline extractor/src/main/resources/list-of-projects.properties:29]: revert

**Review by romani — CHANGES_REQUESTED** [review]
please update /all-projects.properties too

items:

**Pull request closed without merging**